shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -35,7 +43,28 @@
  */
 @ThreadSafe
 public final class DistributedLoadCommand extends AbstractFileSystemCommand {
-  private static final String REPLICATION = "replication";
+  public static final Option REPLICATION =
+      Option.builder()
+          .longOpt("replication")
+          .required(false)
+          .hasArg(true)
+          .desc("number of replicas to have for each block of the loaded file")
+          .build();
+  public static final Option THREAD_OPTION =
+      Option.builder()
+          .longOpt("thread")
+          .required(false)
+          .hasArg(true)
+          .numberOfArgs(1)
+          .argName("threads")
+          .type(Number.class)
+          .desc("Number of mThreads used to load files in parallel, default value is CPU cores * 2")
+          .build();
+
+  private ThreadPoolExecutor mDistributedLoadServicePool;
+  private CompletionService<AlluxioURI> mDistributedLoadService;
+  private ArrayList<Future<AlluxioURI>> mFutures = new ArrayList<>();
```

Comment:
Does a thread safe data structure need to be used here?

Reply:
There is no concurrency here. distributedLoad just load all the file and directory in the specified path orderly and recursively.

Reply:
You're right, I didn't realize that only the RPC call is parallelized.